README.md

```diff
@@ -179,7 +184,21 @@ docker-compose up --build
 To log in to the system once it has been locally deployed, enter a username in the designated field and click the
  "Login" button as seen in the figure below.
 
-#### Users & user types
+#### Authentication
```

Comment:
A suggested reworking of the Authentication description can be found in **bold** here:

[https://www.wrike.com/open.htm?id=1040328943](https://www.wrike.com/open.htm?id=1040328943)